sql/core/src/test/scala/org/apache/spark/sql/execution/joins/BroadcastJoinSuite.scala

```diff
@@ -425,6 +425,29 @@ abstract class BroadcastJoinSuiteBase extends QueryTest with SQLTestUtils
     }
   }
 
+  test("SPARK-33933: AQE broadcast should not timeout with slow map tasks") {
```

Comment:
We should put the test in `AdaptiveQueryExecSuite`

Reply:
Done